CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing
+
+## Rules
+
+Please observe the following ground-rules when making contributions:
+
+-   **All** contributions must be made by opening a pull request into `main`
+-   Please add a description to the pull request describing _what_ features
+    you're implementing or issues you are solving and _how_; add background
+    information whenever possible
+-   Please link all issues and other GitHub items that are related to the
+    changes that you're making
+-   Commit summaries must use the imperative, i.e. "Fix typo" instead of "Fixed
+    typo" or "Fixes typo"
+-   Avoid force pushes when a pull request is in review
+-   A pull request may only be merged when it has received at least one approval
+    from a member of [zeitgeistpm] and GitHub Actions have passed successfully;
+    whenever possible, the author should execute the merge
+-   Comments in a review should be resolved by the author
+-   Please use _squash and merge_ to merge pull requests; feel free to remove
+    trivial items like "Fix typo", etc. from the commit summary
```

Comment:
I don't think anybody should do the merges, it should be someone for the organization. I would not even allow merging for people outside the org (might be even the default setting that is used currently).

Reply:
We might find a more web3 approach in the future, but for now someone from the org should approve and initiate the merge.

Reply:
I must admit that the phrasing is a bit confusing. "Whenever possible, the author should execute the merge" means:

- If the author is an organization member, they should do the merge
- If the author is not an organization member, they _can't_ do the merge because GitHub won't let them